pom.xml

```diff
@@ -82,6 +82,15 @@
             <version>3.2.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.kubernetes</groupId>
+            <artifactId>client-java</artifactId>
+            <version>8.0.2</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-storage</artifactId>
+        </dependency>
```

Comment:
Why these dependencies are in the root pom.xml. They can be easily moved to the pom.xml of the plugin

Reply:
Fixed in e3988a60.